components/proxy/conf/Caddyfile

```diff
@@ -50,16 +50,9 @@
 (enable_log) {
 	log {
 		output stdout
-		format filter {
-			wrap json
-			fields {
-				logger			delete
-				msg				delete
-				size			delete
-				status			delete
-				resp_headers	delete
-				request			delete
-			}
+		format if "status != 200 && status != 0 && status != 304" jsonselect "{severity:level} {timestamp:ts} {logName:logger} {httpRequest>requestMethod:request>method} {httpRequest>protocol:request>proto} {httpRequest>status:status} {httpRequest>responseSize:size} {httpRequest>userAgent:request>headers>User-Agent>[0]} {httpRequest>requestUrl:request>uri}" {
```

Comment:
@leodido maybe we can just use `status > 399`?

Reply:
if that's what we want I can implement the `>` operator and do it this way, ofc :)

Reply:
@leodido let's do that in a future PR :)